### PR TITLE
Show mutated/unconfirmed Oracle in Bet dashboard

### PR DESCRIPTION
**Pull request by dwalintukan**
image

image

### CODE REVIEW

**Comment by liviavinci** [comment]
Please add a pending info label

**Comment by dwalintukan** [comment]
@liviavinci do you mean on the EventCard?

**Comment by liviavinci** [comment]
It would be nice if we have it both on the card and the detail page

**Comment by dwalintukan** [comment]
@liviavinci Added chip and changed text in detail page.

image

File: src/components/EventCard/index.js [hunk 1]
```diff
@@ -14,6 +15,22 @@ import styles from './styles';
 import { getLocalDateTimeString, getEndTimeCountDownString } from '../../helpers/utility';
 
 class EventCard extends React.PureComponent {
+  static propTypes = {
```
Comment (liviavinci) [inline src/components/EventCard/index.js]: why static?

File: src/languageProvider/locales/en_US.json [hunk 2]
```diff
@@ -57,6 +57,7 @@
   "oracle.betStartTimeDisabledText": "The betting start time has not started yet.",
   "oracle.consensus": "Consensus Threshold {value}. This value indicates the amount of BOT needed to set the result.",
   "oracle.enterAmountDisabledText": "You have not entered a valid amount.",
+  "oracle.eventUnconfirmed": "This created Event is unconfirmed. You cannot interact with it until it is confirmed by the blockchain.",
```
Comment (liviavinci) [inline src/languageProvider/locales/en_US.json]: Please add corresponding vars in chinese json

File: src/components/EventCard/index.js (hunk 1 above)
Comment (dwalintukan) [reply src/components/EventCard/index.js]: propTypes are static items as they are all declared the same way across all instances. the previous method of declaring them is the same way:
EventCard.propTypes is declaring them in a static way.

File: src/languageProvider/locales/en_US.json (hunk 2 above)
Comment (dwalintukan) [reply src/languageProvider/locales/en_US.json]: ran the script

**Pull request merged**